### PR TITLE
Don't refer to people as "other things"

### PR DESCRIPTION
**Pull request by dwsinger**
etc. means "and other things", so we need et al., or more likely the english "and others"

### CODE REVIEW

**Review by tantek — CHANGES_REQUESTED** [review]
"etc." seems less "othering" than explicitly saying *others*.

Also, while I don't think "etc." implies "other things", if we want more precision, we could use "et al." instead per https://en.wikipedia.org/wiki/Et_cetera#Similar_Latin_expressions. Ah I see you already suggested "et al". Yes That would be better than "and others" which sounds more explicitly "othering" which I think we want to avoid.

**Comment by dwsinger** [comment]
et al. is indeed the correct latin for "and other people" but it's a little archaic and not widely used/understood. see [wikipedia](https://en.wikipedia.org/wiki/Et_cetera) "cetera" is neuter plural and means "things"; it's generally thought rude to use it to refer to people, and since et alii is rare and not so easily understood, I thought "and others". But maybe we shouldn't be vague here; it we mean other classes, name them, or it just looks like a lazy catch-all?

**Comment by tantek** [comment]
> et al. is indeed the correct latin for "and other people" but it's a little archaic and not widely used/understood. 

I'm going to firmly disagree in the context of W3C — et al is commonly (nearly always?) used in Normative and Informative references in specifications, e.g. https://www.w3.org/TR/css-color-3/#normative

> see [wikipedia](https://en.wikipedia.org/wiki/Et_cetera) "cetera" is neuter plural and means "things"; it's generally thought rude to use it to refer to people,

I feel like I’ve both never heard that that usage is rude, and have seen it commonly done without any offense. Curious if that’s a historical rudeness that’s perhaps faded away or if there is a specific culture within which it is considered rude.

> and since et al is rare and not so easily understood,
Never heard anyone complain about its usage in spec references so I think that's a non-issue in the context of W3C.

My problem with the use of "et al" here is that the only contexts I've seen it used all imply a discrete list of people, rather than an open ended expression, and this context (this document) intends it open endedly, in contrast to the use of spec references which are a discrete list of people.

> I thought "and others". But maybe we shouldn't be vague here; it we mean other classes, name them, or it just looks like a lazy catch-all?

No, quite the opposite, a catch-all is way of humbly admitting that there are unknown unknown sets of people who we don’t want to exclude by lack of explicit inclusion, and not committing to any attempt to explicitly "name" a bunch of "other classes" which may be an impossible task as such a list could grow in the future (as many of these terms didn't exist decades ago).

**Comment by frivoal** [comment]
I think "etc." is fine. If you don't want to use Latin, use "and so on" or simply an ellipsis (…).

"Etc." is well known, and in use in numerous modern languages. "Et al." much less so.  For instance, "etc." is well established in French, but I had never heard "et al." prior to seeing it in research paper bibliographies, and I could not find it in French language dictionaries. Same for Japanese.

Besides, (some?) English dictionaries say "etc." is fine for people as well. See for instance [Merriam Webster](https://www.merriam-webster.com/dictionary/et%20cetera):

> a number of unspecified additional persons or things

I vote for a simple "…"

--------
_Skip the rest if you're not interested in Latin._





In Latin, "et cetera" means "and other", or "and the rest" and because of the neutral gender, does indeed imply "and other [things]". At the same time, "et ceteri" would be perfectly fine Latin to talk about people, is attested, and could plausibly be abbreviated to "etc." as well, although that didn't become formulaic in English. "et alii" would be valid Latin grammar, but it isn't established as a set phrase in Latin. (See https://latin.stackexchange.com/questions/13822/et-cetera-versus-et-alia)

Besides, strictly speaking, "cetera" means more "the rest" or "the remainder", or "further similar things" than "others". The meaning of "alii" is close, but subtlety different (I'd say it's more about referring to alternatives in a known set than to continuation). In a list of co-authors, "et alii" is reasonable, as we're talking about "someone, and their (known, but not enumerated) co-authors". While "et ceterii" would mean "someone, and a bunch more people".

But "et alii" is only about people because of the "ii" ending. "Et alia" would be about things. So if you go by Latin grammar, "et alii" and "et cetera" are not equivalent expressions just with one being for people and one being for things. The direct equivalent of "et cetera" for people would be "et ceteri".

But while Latin is fun, I don't think any of this matters. We're not speaking Latin here. In modern languages, "etc." is fine for people, and "et al." is much less understood. And "…" works fine.

File: Vision/README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ In our operations and membership, we will:
 *	Provide an open forum, where diverse voices from around the world and industries come together, incubate and build consensus for global standards for Web technologies, including organizations of all sizes (from single-person companies to multi-nationals) and considering public feedback.
 * Encourage incubation in new areas and industries with open platforms for discussion, collaboration and innovation, making it more structured and improving consensus-building among key stake holders.
 * Strive for the broadest participation, along axes including worldwide participation, diversity and inclusion, facilitating balance, equity and cooperation among the participants from different industries, user groups and organizational sizes, and thus establishing W3C as representative of the whole community.
-* Increase involvement of under-represented key stakeholders such as end users, content creators, developers etc.
+* Increase involvement of under-represented key stakeholders such as end users, content creators, developers, and others.
```
Comment (dennis-dingwei) [inline Vision/README.md:44]: Agree with "others"  as seems to me more formal than "etc".

**Comment by dwsinger** [comment]
The other way to make it open-ended is to say "under-represented, including X, Y, Z" which is then not a closed set.

**Review by tantek — CHANGES_REQUESTED** [review]
@dwsinger said "shouldn’t be vague here" and in re-reading this in context, I’ve come to agree with that assessment, because this is about "key stakeholders". 

If we can’t name a "key stakeholder" we shouldn’t pretend that there is some sort of open-ended judgment call. I suggest dropping any kind of "etc. or others" and rewording that trailing list as:

"end users, content creators, and developers."

The "such as" already makes it clear that that list is not exhaustive, so there is no semantic need for a clause like etc. or "others".

**Comment by cwilso** [comment]
I see at least six options here, listed in my personal preference order:

1. Remove "etc." - the sentence already reads "such as end users, content creators, [and] developers" - which is open-ended.
2. Change it to "et al".
3. Change it to "and others".
4. Change it to mirror the previous similar occurrence in the doc "and more".
5. Change it to "including" i.e. "including end users, content creators, [and] developers"
6. Leave it as "etc."

I would propose choice 1.

**Comment by cwilso** [comment]
In our 15 December 2022 meeting, the AB resolved to simply remove "etc.".  I filed #32 to track this, and will fix it once #30 lands.)

**Pull request closed without merging**